### PR TITLE
Fix/count agg function empty or fails

### PR DESCRIPTION
**Pull request by ideoma**

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 76 / 90 (84.44%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|3|8|37.50%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|4|10|40.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|4|5|80.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|6|7|85.71%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|17|94.12%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/lt/LtIntFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/groupby/GroupByUtils.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 89 / 90 (98.89%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|9|10|90.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/lt/LtIntFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/groupby/GroupByUtils.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 90 / 90 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/lt/LtIntFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/groupby/GroupByUtils.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 93 / 93 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/lt/LtIntFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

File: core/src/test/java/io/questdb/cairo/RecordCursorPrinter.java [hunk 1]
```diff
@@ -160,5 +166,8 @@ protected void printColumn(Record r, RecordMetadata m, int i, CharSink sink) {
             default:
                 break;
         }
+        if (printTypes) {
+            sink.put(":").put(ColumnType.nameOf(m.getColumnType(i)));
```
Comment (bluestreak01) [inline core/src/test/java/io/questdb/cairo/RecordCursorPrinter.java:170]: can be ':'

File: core/src/main/java/io/questdb/griffin/engine/groupby/vect/SumTimestampVectorAggregateFunction.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ public void aggregate(long address, long addressSize, int workerId) {
     }
 
     @Override
-    public void aggregate(long pRosti, long keyAddress, long valueAddress, long valueAddressSize, int workerId) {
+    public void aggregate(long pRosti, long keyAddress, long valueAddress, long valueAddressSize, int columnSizeHint, int workerId) {
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/griffin/engine/groupby/vect/SumTimestampVectorAggregateFunction.java:72]: parameter name continuity

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 92 / 92 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/lt/LtIntFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|16|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 91 / 91 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|16|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 125 / 125 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|18|18|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|20|20|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|16|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 117 / 117 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|15|15|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|15|15|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|16|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 125 / 125 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunction.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountLong256GroupByFunction.java|18|18|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunction.java|20|20|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunction.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountSymbolGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/MaxFloatGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/ColumnType.java|16|16|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstFloatGroupByFunction.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/GroupByFunction.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/FirstCharGroupByFunction.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/LastCharGroupByFunctionFactory.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountStringGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/IntFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|2|2|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**